CogniteSdk.Types/Alpha/Simulators/SimulatorValue.cs

```diff
@@ -114,5 +136,83 @@ public override bool Equals(object obj)
             /// </summary>
             public override int GetHashCode() => Value.GetHashCode();
         }
+
+        /// <summary>
+        /// Simulator value that contains a list of string values.
+        /// </summary>
+        public sealed class StringArray : SimulatorValue
+        {
+            /// <summary>
+            /// Initialize new string list value.
+            /// </summary>
+            /// <param name="value">The value to set.</param>
+            public StringArray(IEnumerable<string> value)
+            {
+                Type = SimulatorValueType.STRING_ARRAY;
+                Value = value;
+            }
+
+            /// <summary>
+            /// The contained string array value.
+            /// </summary>
+            public IEnumerable<string> Value { get; set; }
+
+            /// <summary>
+            /// Return string representation of the string array value, i.e the string array itself.
+            /// </summary>
+            public override string ToString() => string.Join(", ", Value);
+
+            /// <summary>
+            /// Check if two string array values are equal.
+            /// </summary>
+            public override bool Equals(object obj)
+            {
+                return obj != null && Value.SequenceEqual((obj as StringArray).Value);
+            }
+
+            /// <summary>
+            /// Get hash code for the string array value.
+            /// </summary>
+            public override int GetHashCode() => Value.GetHashCode();
+        }
+
+        /// <summary>
+        /// Simulator value that contains a list of double values.
+        /// </summary>
+        public sealed class DoubleArray : SimulatorValue
+        {
+            /// <summary>
+            /// Initialize new double list value.
+            /// </summary>
+            /// <param name="value">The value to set.</param>
+            public DoubleArray(IEnumerable<double> value)
+            {
+                Type = SimulatorValueType.DOUBLE_ARRAY;
+                Value = value;
+            }
+
+            /// <summary>
+            /// The contained double array value.
+            /// </summary>
+            public IEnumerable<double> Value { get; set; }
```

Comment:
This can be made readonly if this is only set in the constructor once and then not changed at all 

Something like: 
`public IReadOnlyList<double> Value { get; }`

Reply:
made setter private now `Value { get; private set; }`, I guess that resolves the concern